internal/npm_install/generate_build_file.js

```diff
@@ -768,7 +768,7 @@ node_module_library(
     # ${pkg._dir} package contents (and contents of nested node_modules)
     srcs = [
         ":${pkg._name}__files",
-    ],${depsStarlark}
+    ],${depsStarlark}${scriptStarlark}
```

Comment:
Duplicating the scripts in __files and __pkg is more correct than having the transitive

```
        if NodeModuleSources in src:
            scripts = depset(transitive = [scripts, src[NodeModuleSources].scripts])
```

lookup in `node_module_library.bzl` since __files is mainly a construct for avoiding circular deps and it should not be generally used.